front/lib/auth.ts

```diff
@@ -346,10 +367,12 @@ export class Authenticator {
       throw new Error(`Could not find workspace with sId ${workspaceId}`);
     }
 
+    let globalGroup: GroupResource | null = null;
```

Comment:
`internalBuilderForWorkspace` will likely have to go away or change as some of its use assume ability to operate on any group I believe?

In any case not for now but maybe leave a `// TODO(GROUPS_INFRA)` that this is not group compatible anymore and will have to be refactored?